app/app_test.go

```diff
@@ -64,6 +65,23 @@ func (iter *JSONIterator) Close(ctx context.Context) error {
 	return nil
 }
 
+func TestNew(t *testing.T) {
+	app := New(nil, nil, nil)
+	app = app.WithWebhooksTimeout(10)
+
+	assert.NotNil(t, app)
+}
+
+func TestRunAndLogError(t *testing.T) {
+	ctx := context.Background()
+	run := false
+	runAndLogError(ctx, func() error {
+		run = true
+		return errors.New("error")
+	})
+	assert.True(t, run)
```

Comment:
This test is hardly making an assertion about the behavior of `runAndLogError`, could you catch the log and add an additional assertion that the log is actually caught?
```suggestion
	var b bytes.Buffer
	logger := log.NewEmpty()
	logger.Log.Output = &b
	ctx = log.WithContext(ctx, logger)	
	run := false
	const errString = "unique string caught by logger"
	runAndLogError(ctx, func() error {
		run = true
		return errors.New(errString)
	})
	assert.True(t, run)
	assert.Contains(t, buf.String(), errString)
```